influxdb/content.md

```diff
@@ -6,32 +6,395 @@ InfluxDB is a time series database built from the ground up to handle high write
+
+Examples:
+
+```console
+# Config setting: bolt-path
+INFLUXD_BOLT_PATH=/root/influxdb.bolt
+# Config setting: no-tasks
+INFLUXD_NO_TASKS=true
+# Config setting: storage-wal-fsync-delay
+INFLUXD_STORAGE_WAL_FSYNC_DELAY=15m
+```
+
+Finally, all config options can be passed as CLI options:
+
+```console
+$ docker run -p 8086:8086 \
+      %%IMAGE%% --storage-wal-fsync-delay=15m
+```
+
+CLI options take precedence over environment variables.
+
+Find more about configuring InfluxDB [here](https://docs.influxdata.com/influxdb/v2.0/reference/config-options/).
+
+### Database Setup
+
+InfluxDB 2.0 requires authentication. A special API exists to bootstrap the first super-user in the database, along with an initial organization and bucket. It's possible to access this API manually, or to run it automatically via environment variables.
+
+#### Manual Setup
+
+If your InfluxDB container is running locally (or on a host exposed to the network), you can perform initial setup from outside the container using either the UI or the `influx` CLI. Find more about setting up InfluxDB using these methods [here](https://docs.influxdata.com/influxdb/v2.0/get-started/#set-up-influxdb).
+
+It's also possible to perform manual setup from within the container using `docker exec`. For example, if you start the container:
+
+```console
+$ docker run -d -p 8086:8086 \
+      --name influxdb2 \
+      -v $PWD:/var/lib/influxdb2 \
+      %%IMAGE%%
+```
+
+You can then run the `influx` client in the container:
+
+```console
+$ docker exec influxdb2 influx setup \
+      --username $USERNAME \
+      --password $PASSWORD \
+      --org $ORGANIZATION \
+      --bucket $BUCKET
+```
+
+Running setup from within the container will cause CLI configs to be written to `/etc/influxdb2/influx-configs`. You can then use the `influx` CLI from within the container to extract the generated admin token:
+
+```console
+# Using table output + cut
+$ docker exec influxdb2 influx auth list \
+      --user $USERNAME \
+      --hide-headers | cut -f 3
+
+# Using JSON output + jq
+$ docker exec influxdb2 influx auth list \
+      --user $USERNAME \
+      --json | jq -r '.[].token'
+```
+
+Alternatively, you could configure your initial InfluxDB run to mount `/etc/influxdb2` as a volume:
+
+```console
+$ docker run -d -p 8086:8086 \
+      --name influxdb2 \
+      -v $PWD/data:/var/lib/influxdb2 \
+      -v $PWD/config:/etc/influxdb2 \
+      %%IMAGE%%
+```
+
+This will make the generated CLI configs available to the host.
+
+#### Automated Setup
+
+The InfluxDB image contains some extra functionality to automatically bootstrap the system. This functionality is enabled by setting the `INFLUXD_INIT_MODE` environment variable to the value `setup` when running the container. Additional environment variables are used to configure the setup logic:
+
+-	`INFLUXD_INIT_USERNAME`: The username to set for the system's initial super-user (**Required**).
```

Comment:
Caught by an early adopter: all of these env variables need corrected, the prefix is `INFLUXDB_`. Will fix